asset-transfer-secured-agreement/chaincode-go/README.md

```diff
@@ -1,342 +0,0 @@
-# Secured private asset transfer scenario
```

Comment:
I would keep this REAMDE, but just include a link to the tutorial. That would provide some instructions just in case users do not land on the front REAMDE.